samtranslator/model/api/http_api_generator.py

```diff
@@ -88,6 +102,71 @@ def _construct_http_api(self):
 
         return http_api
 
+    def _add_cors(self):
+        """
+        Add CORS configuration if CORSConfiguration property is set in SAM.
+        Adds CORS configuration only if DefinitionBody is present and
+        APIGW extension for CORS is not present in the DefinitionBody
+        """
+
+        if not self.cors_configuration:
+            return
+
+        if self.cors_configuration and not self.definition_body:
+            raise InvalidResourceException(
+                self.logical_id, "Cors works only with inline OpenApi specified in " "'DefinitionBody' property"
```

Comment:
period at the end of the error message. Also, there are double quotes in there, by accident? `" "'DefinitionBody'`

Reply:
I was using an older version of black, I think black has reformatted that. I will check and update the quotes and periods.